content/docs/setup/customize/aws-elb/alb.md

```diff
@@ -0,0 +1,185 @@
+---
+title: AWS ALB
+weight: 10
+---
+
+In this guide you explore how to expose the {{< reuse "docs/snippets/product-name.md" >}} proxy with an AWS application load balancer (ALB). 
+
+{{< callout type="warning" >}}
+The AWS Load Balancer Controller only supports creation of an ALB through an Ingress Controller and not through the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}}. Because of this, you must create the ALB separately through an Ingress resource that connects it to the service that exposes your gateway proxy.
```

Comment:
```suggestion
The AWS Load Balancer Controller only supports the creation of an ALB through an Ingress Controller and not through the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}}. Because of this, you must create the ALB separately through an Ingress resource that connects it to the service that exposes your gateway proxy.
```